script.js

```diff
@@ -0,0 +1,86 @@
+// variables
+const divide = document.getElementById('/')
+const multiply = document.getElementById('*')
+const minus = document.getElementById('-')
+const plus = document.getElementById('+')
+const equals = document.getElementById('=')
+const reset = document.getElementById('reset')
+const numberButtons = document.querySelectorAll('.number-button')
+const displayEl = document.getElementById('calc-display')
+
+// state
+let val1 = 0;
+let val2 = 0;
+let currentOperation = null;
+
+document.addEventListener("DOMContentLoaded", function () {
+    // make sure buttons have values based on thier id
+    function setNumberButtonValues() {
```

Comment:
i see what you are trying to do, but ultimately any time you read a value from  an El on the DOM you will get a string. 

i feel like you could have just saved this step, and just did the conversion of the id itself isf it is not NaN